cmd/tailscale/cli/up.go

```diff
@@ -82,6 +82,25 @@ func acceptRouteDefault(goos string) bool {
 	return p.DefaultRouteAll(goos)
 }
 
+// singleUseStringFlag will throw an error if the flag is specified more than once.
+type singleUseStringFlag struct {
+	set   bool
+	value string
+}
+
+func (s singleUseStringFlag) String() string {
+	return s.value
+}
+
+func (s *singleUseStringFlag) Set(v string) error {
+	if s.set {
+		return fmt.Errorf("flag can only be specified once")
```

Comment:
Nit: since nothing is being formatted, you could use `errors.New` here.

Reply:
Good call 👍